### PR TITLE
[SW-609] UX/UI improvements of Account selection modal

### PR DESCRIPTION
**Pull request by konradodo**

### CODE REVIEW

**Review by peronczyk — APPROVED** [review]

File: src/popup/components/AccountSelectOptionsItem.vue [hunk 1]
```diff
@@ -89,7 +89,7 @@ export default defineComponent({
     @include mixins.flex(space-between, flex-start, row);
 
     position: relative;
-    padding: 8px;
+    padding: 5px 8px;
```
Comment (CedrikNikita) [inline src/popup/components/AccountSelectOptionsItem.vue:92]:
```suggestion
    padding: 6px 8px;
```
Judging by element size in design. 
image
Current size 
image

File: src/popup/components/MultisigVaultCreateReview.vue [hunk 2]
```diff
@@ -208,8 +208,9 @@ export default defineComponent({
   gap: var(--gap);
 
   .creator-error-message {
-    @extend %face-sans-14-regular;
```
Comment (CedrikNikita) [inline src/popup/components/MultisigVaultCreateReview.vue:211]: But in figma the size is `14px` for this element.

Comment (konradodo) [reply src/popup/components/MultisigVaultCreateReview.vue:211]: 14 px is only for amount value
image

**Review by CedrikNikita — APPROVED** [review]

File: src/popup/components/MultisigVaultCreateReview.vue (hunk 2 above)
Comment (konradodo) [reply src/popup/components/MultisigVaultCreateReview.vue:211]: fixed

**Pull request merged**